src/Julia/swapTwoNumbers.java

```diff
@@ -0,0 +1,26 @@
+package Julia;
+
+import java.sql.SQLOutput;
+
+public class swapTwoNumbers {
+    //Swap two int variables' values without using a third variable
+
+    public static void main(String[] args) {
+
+        int num1 = 50;
+        int num2 = 100;
+
+        System.out.println("Num1 Value before swapping: " + num1);
+        System.out.println("Num2 Value before swapping: " + num2);
+
+        num1 = num2 + num1; // 50 + 100 = 150
+        num2 = num1 - num2; //
```

Comment:
Did you forget to add a comment here?

Reply:
Yes I already fixed it:)  Let me send you another pull request!